examples/jsm/renderers/webgl/nodes/GLSLNodeBuilder.js

```diff
@@ -51,12 +51,16 @@ class GLSLNodeBuilder extends NodeBuilder {
 
 	getPropertyName( node, shaderStage ) {
 
-		if ( node.isOutputStructVar ) return '';
-
 		return super.getPropertyName( node, shaderStage );
 
 	}
 
+	getOutputName() {
```

Comment:
What do you think `getOutputStructName()`?

Reply:
sounds good, I'll change it